pillar.example

```diff
@@ -77,6 +77,9 @@ nginx:
       # nginx.conf (main server) declarations
       # dictionaries map to blocks {} and lists cause the same declaration to repeat with different values
       config: 
+        source: salt://path_to_nginx_conf_file/nginx.conf # IMPORTANT: This option is mutually exclusive with the rest of the
```

Comment:
should this be `source_path`

Reply:
What do you mean, replace ``source`` for ``source_path`` as the name of the option in the pillar?